src/Storages/MergeTree/MergeTreeIndexUSearch.cpp

```diff
@@ -41,22 +41,37 @@ namespace ErrorCodes
 namespace
 {
 
+std::unordered_map<String, unum::usearch::metric_kind_t> nameToMetricKind = {
```

Comment:
`const` for president! I think here it actually makes sense, because we don't want to change this map.